src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -377,118 +377,15 @@
-     * {@link org.spongepowered.api.data.Property.Operator} of
-     * {@link org.spongepowered.api.data.Property.Operator#EQUAL}.
-     *
-     * @param props inventory properties to query for
-     * @param <T> expected inventory type, specified as generic to allow easy
-     *      pseudo-duck-typing
-     * @return the query result
-     */
-    <T extends Inventory> T query(InventoryProperty<?, ?>... props);
-
-    /**
-     * Query this inventory for inventories matching any of the supplied titles.
-     * Logical <code>OR</code> is applied between operands.
-     *
-     * @param names the names of the inventories to search for
-     * @param <T> expected inventory type, specified as generic to allow easy
-     *      pseudo-duck-typing
-     * @return the query result
-     */
-    <T extends Inventory> T query(Translation... names);
-
-    /**
-     * Query this inventory for inventories matching any of the supplied titles.
-     * Logical <code>OR</code> is applied between operands.
-     *
-     * @param names the names of the inventories to search for
-     * @param <T> expected inventory type, specified as generic to allow easy
-     *      pseudo-duck-typing
-     * @return the query result
-     */
-    <T extends Inventory> T query(String... names);
-
-    /**
-     * <p>Query this inventory by dynamically inspecting each operand. Each
-     * operand in turn is checked for a match against the other query methods,
-     * and if a matching method is found the query is performed using the
-     * operand. This is repeated until all operands are consumed and allows a
-     * union of multiple query types to be aggregated into a single view.</p>
-     *
-     * <p>For operands with no matching type, the behaviour is determined by the
-     * individual inventory. A naive match may be obtained by calling .equals()
-     * against the child inventory passing the unknown operand as an argument.
-     * </p>
-     *
-     * @param args search parameters
-     * @param <T> expected inventory type, specified as generic to allow easy
-     *      pseudo-duck-typing
-     * @return the query result
-     */
-    <T extends Inventory> T query(Object... args);
-
-    /**
-     * Query this inventory for inventories containing any stacks which match
-     * the supplied stack operands ignoring its quantity. This query operates
-     * directly on {@link Slot} leaf nodes in the inventory and will always
-     * return a collection containing only {@link Slot} instances. Logical
-     * <code>OR</code> is applied between operands.
-     * This ignores stack sizes. To query for stacks of a specific size use
-     * {@link #query(ItemStack...)}.
-     *
-     * @param types items to query for, the size of the stacks is always ignored
-     * @param <T> expected inventory type, specified as generic to allow easy
-     *      pseudo-duck-typing
-     * @return the query result
-     */
-    <T extends Inventory> T queryAny(ItemStack... types);
```

Comment:
I got most of them, except for two: the `Translation` one and the `String` one. The `String` one wasn't even implemented, but I'll get the `Translation` one tomorrow.